roles/ffh.mesh_wireguard_remotes_peers_git/templates/autoupdate_wg.py.j2

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import subprocess
+from calc import calc_ll
+from pyroute2 import WireGuard
+
+wgintf_base="wgtest-"
+
+REPO="/etc/wireguard/peers-wg"
+wg_domains=[10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 99]
```

Comment:
As you already added the ".j2" file ending, I would suggest to use jinja to generate this line.

Reply:
jop

Reply:
I think its fixed in 771a5d0c51a4d1c5601524046f36887c7740d3dd